task_14/Vanya.Ryabchenko/src/test/java/com/hillel/geometry/GeometryTest.java

```diff
@@ -0,0 +1,54 @@
+package com.hillel.geometry;
+
+
+import static org.hamcrest.core.Is.is;
+import static org.junit.Assert.assertThat;
+
+
+import org.junit.Before;
+import org.junit.Test;
+
+public class GeometryTest {
+
+  Figure rectangle;
+  Figure square;
+  Figure circle;
+
+  @Before
+  public void setUp() {
+    rectangle = new Rectangle(2, 3);
+    square = new Square(5);
+    circle = new Circle(3);
+  }
+
+  @Test
+  public void rectangleSquareTst() {
+    assertThat(rectangle.square(), is(6.0));
+  }
+
+  @Test
+  public void rectanglePerimeterTest() {
+    assertThat(rectangle.perimeter(), is(10.0));
+  }
+
+  @Test
+  public void squareSquareTest() {
+    assertThat(square.square(), is(25.0));
+  }
+
+  @Test
+  public void squarePerimeterTest() {
+    assertThat(square.perimeter(), is(20.0));
+  }
+
+  @Test
+  public void circleSquareTest() {
+    assertThat(circle.square(), is(28.274333882308138));
```

Comment:
You can use isCloseTo here:
assertThat(circle.square(), is(closeTo(28.27, 0.01));
if you want to avoid writing all the digits after decimal point